public_html/wp-content/plugins/wordcamp-speaker-feedback/views/form-feedback.php

```diff
@@ -6,65 +6,105 @@
 ?>
 <hr />
 <form id="sft-feedback" class="speaker-feedback">
-	<h3><?php esc_html_e( 'Rate this talk', 'wordcamporg' ); ?></h3>
-	
+
+	<h3><?php esc_html_e( 'Leave Feedback', 'wordcamporg' ); ?></h3>
+
+	<?php if ( ! is_user_logged_in() ) : ?>
 	<div class="speaker-feedback__field">
-		<fieldset class="speaker-feedback__field-rating" aria-label="<?php esc_attr_e( 'Rate this talk', 'wordcamporg' ); ?>">
+		<div class="speaker-feedback__notice">
+			<p><?php echo wp_kses_post( sprintf(
+				__( '<a href="%s">Log in to your WordPress.org account,</a> or add your name & email to leave feedback.', 'wordcamporg' ),
+				wp_login_url( get_permalink() . 'feedback' )
+			) ); ?></p>
+		</div>
+
+		<div class="speaker-feedback__field-inline">
+			<label for="sft-author-name">
+				<?php esc_html_e( 'Name', 'wordcamporg' ); ?>
+			</label>
+			<input type="text" id="sft-author-name" name="sft-author-name" required />
+		</div>
+
+		<div class="speaker-feedback__field-inline">
+			<label for="sft-author-email">
+				<?php esc_html_e( 'Email', 'wordcamporg' ); ?>
+			</label>
+			<input type="email" id="sft-author-email" name="sft-author-email" required />
+		</div>
+	</div>
+	<?php else : ?>
+		<div class="speaker-feedback__notice">
+			<p><?php
+			$user = wp_get_current_user();
+			echo wp_kses_post( sprintf(
+				__( 'You are leaving feedback as %s.', 'wordcamporg' ),
+				$user->display_name
+			) ); ?></p>
+		</div>
+	<?php endif; ?>
+
+	<div id="speaker-feedback-notice" aria-live="polite" aria-relevant="additions text" aria-atomic="true"></div>
+
+	<div class="speaker-feedback__field">
+		<fieldset class="speaker-feedback__field-rating">
+			<legend><?php esc_html_e( 'Rate this talk', 'wordcamporg' ); ?></legend>
```

Comment:
Can we populate this label from `CommentMeta\get_feedback_questions()`? And also conditionally include the `required` attribute on the inputs via `CommentMeta\get_feedback_meta_field_schema()`?